docs/.vitepress/config.ts

```diff
@@ -0,0 +1,147 @@
+import path from 'path';
+import { writeFileSync } from 'fs';
+import { Feed } from 'feed';
+import { createContentLoader, defineConfig, type SiteConfig } from 'vitepress';
+import deflist from 'markdown-it-deflist';
+import footnote from 'markdown-it-footnote';
```

Comment:
Need to tweak the tsconfig setup so these resolve. Probably what we actually need is a workspace config.